packages/appstore/src/components/wallets-banner/__tests__/wallets-banner.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { isMobile } from '@deriv/shared';
+import WalletsBannerUpgrade from '../wallets-banner-upgrade';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
+    // isDesktop: jest.fn(() => true),
```

Comment:
Please remove the comments if they are not needed. 🙏

Reply:
@hamid-deriv agree, my fault